docs/customization.md

```diff
@@ -254,17 +280,46 @@ python -m venv venv
 source venv/bin/activate
 ```
 
+!!! note "Get pip to refuse working outside a `venv`"
```

Comment:
Very good idea, but I think the headline is a little misleading. Maybe something like:

Ensure pip always runs in a virtual environment